source/Cosmos.HAL2/Global.cs

```diff
@@ -67,6 +68,8 @@ static public void Init(TextScreenBase textScreen)
             mDebugger.Send("Network Devices Init");
             NetworkInit.Init();
 
+            AudioManager.Init();
```

Comment:
Could you send some log through the debugger?
mDebugger.Send("Audio Manager Init"); or something